valimail.com.valimail-authenticate.json

```diff
@@ -0,0 +1,38 @@
+{
+  "providerId": "valimail.com",
+  "providerName": "Valimail",
+  "serviceId": "valimail-authenticate",
+  "serviceName": "Valimail Authenticate",
+  "version": 1,
+  "logoUrl": "https://valimail-logos.s3.amazonaws.com/valimail-logo-primary.svg",
+  "description": "Enable Valimail Authenticate on a domain",
+  "variableDescription": "%spftxt% is the SPF record",
+  "records": [
+      {
+      "type": "NS",
+      "host": "_dmarc",
+      "pointsTo": "ns.vali.email",
+      "ttl": 3600
+    },
+    {
+      "type": "NS",
+      "host": "_domainkey",
+      "pointsTo": "ns.vali.email",
+      "ttl": 3600
+    },
+    {
+      "type": "NS",
+      "host": "_bimi",
```

Comment:
same comment as for _domainkey, this record would block more than one key.

Reply:
Understood, as with DKIM we manage all selectors under the label. (Note: I'm the author of the BIMI spec: https://tools.ietf.org/html/draft-blank-ietf-bimi-01)